plugins/binanceExchange_ws.go

```diff
@@ -398,16 +398,6 @@ func (beWs *binanceExchangeWs) GetOrderBook(pair *model.TradingPair, maxCount in
 	askCcxtOrders := book.Asks
 	bidCcxtOrders := book.Bids
 
-	if fetchLimit != maxCountInt {
```

Comment:
I think there's a misunderstanding from my previous comment. We still need this I think. Just not the outer if `fetchLimit != maxCount` check.

```golang
// we may not have fetched all the requested levels because the exchange may not have had that many levels in depth
if len(askCcxtOrders) > maxCountInt {
    askCcxtOrders = askCcxtOrders[:maxCountInt]
}
if len(bidCcxtOrders) > maxCountInt {
    bidCcxtOrders = bidCcxtOrders[:maxCountInt]
}
```